### PR TITLE
feature(swift5.1): add info about migrating a plugin to Swift 5.1

### PR DESCRIPTION
**Pull request by snooky23**
## Description
Add info about migrating a plugin to Swift 5.1

## Checklist

* [*] PR is scoped to one task
* [ ] Tests are included
* [ ] Documentation is included

* [*] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

File: book/whats_new/20191124-swift-5.1-migration.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Swift 5.1 Migration
+
+This article will go through recent changes in Zapp iOS SDK 13.0 and up made for the swift 5.1 migration. For the full SDK 13.0 changelog go [here](https://github.com/applicaster/Zapp-iOS/blob/development/CHANGELOG.md).
```
Comment (alexzchut) [inline book/whats_new/20191124-swift-5.1-migration.md]: Swift

File: book/whats_new/20191124-swift-5.1-migration.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+
+## Internal Applicaster Plugin Migration
+
+Internal plugins should follow these instructions in order to update the creation of a closed framework.
+
+1. Install xcode 11.1 and cocoapods version 1.7.5 using the following command:
```
Comment (alexzchut) [inline book/whats_new/20191124-swift-5.1-migration.md]: Xcode, Cocoapods

File: book/whats_new/20191124-swift-5.1-migration.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    gem 'cocoapods', '= 1.7.5'
+    gem 'fastlane'
+    ```
+
+3. Update the Gemfile.lock by running `bundle install` in your main plugin directory.
+
+4. Go to circle-ci confog.yml file and change the following:
+    * Change the xcode version to 11.2.1
+    * Change the deploy version to 0.10.0
+    * Increment the caching keys to a new version, for example:
```
Comment (alexzchut) [inline book/whats_new/20191124-swift-5.1-migration.md]: if there is a step:
`name: Ruby - set version`
 `command:  echo "ruby-2.4" > ~/.ruby-version`
need to update to 2.5

File: book/whats_new/20191124-swift-5.1-migration.md (hunk 1 above)
Comment (snooky23) [reply book/whats_new/20191124-swift-5.1-migration.md]: Done

File: book/whats_new/20191124-swift-5.1-migration.md (hunk 2 above)
Comment (snooky23) [reply book/whats_new/20191124-swift-5.1-migration.md]: בוצע

File: book/whats_new/20191124-swift-5.1-migration.md (hunk 3 above)
Comment (snooky23) [reply book/whats_new/20191124-swift-5.1-migration.md]: done

**Pull request merged**